fastlane/lib/fastlane/auto_complete.rb

```diff
@@ -0,0 +1,16 @@
+module Fastlane
+  # Enable tab auto completion
+  class AutoComplete
+    def self.execute
+      path = File.join(Fastlane::Helper.gem_path('fastlane'), 'lib', 'assets', 'completions')
+
+      fastlane_conf_dir = "~/.fastlane"
+      `mkdir -p #{fastlane_conf_dir} && cp -r #{path} #{fastlane_conf_dir}`
+
+      UI.success "To use auto complete for fastlane"
+      UI.success "add the following line to your favorite rc file (e.g. ~/.bashrc)"
+      UI.important "  . ~/.fastlane/completions/completion.sh"
```

Comment:
@mlbileschi said the `.` is more [universal](http://askubuntu.com/questions/25488/what-is-the-difference-between-source-and-in-bash) than `source`

Like the `pbcopy` idea but that would only work on OS X right?